test/Microsoft.Azure.WebJobs.Host.EndToEndTests/InvocationFilterEndToEndTests.cs

```diff
@@ -0,0 +1,637 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
+{
+    public class InvocationFilterEndToEndTests : IClassFixture<TestFixture>
```

Comment:
>InvocationFilterEndToEndTests [](start = 17, length = 29)

We should be able to move these to unit tests and decouple from external things like storage. I can show you how.